include/reduced_basis/rb_parametrized.h

```diff
@@ -27,6 +27,7 @@
 #include "libmesh/reference_counted_object.h"
 
 // C++ includes
+#include <cstddef>
```

Comment:
Was this include needed for `std::size_t` or something?

Reply:
Yeah probably. I run IWYU along with my linter, and it automatically adds these sometimes. It probably compiles without this line, by pulling it in from somewhere els.